src/components/AttachmentViewer/components/InteractiveImage/styled.ts

```diff
@@ -35,4 +37,19 @@ export const ZoomedImage = styled.div.attrs<ZoomedImageProps>(
 )<ZoomedImageProps>`
   will-change: background-position;
   ${({ zoom }) => `background-size: ${zoom * 200}%`}
+  display: inline-block;
+`
+export const Wrapper = styled.div``
+
+export const StyledFigCaption = styled.figcaption`
+  background-color: ${themeColor('tint', 'level1')};
+  height: 100%;
+  font-size: 1.5rem;
+  font-weight: 700;
+  width: 0;
+  min-width: calc(100% - 40px);
+  overflow-wrap: break-word;
+  padding: 20px;
```

Comment:
padding is something we use themespacing for

Reply:
you're right, was a little to quick with the  copy-paste. Changed it.